migration.ts

```
@@ -0,0 +1,118 @@
+import { convertCsvToArray } from "./src/utils/convertCsvToArray";
+import { generateMigration } from "./src/migrationConvertLogic";
+import { convertTextToPhpFile } from "./src/utils/convertTextToPhpFile";
+import * as fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)